modules/UI/recording/Recording.js

```diff
@@ -302,7 +302,13 @@ const Recording = {
         if (config.iAmRecorder) {
             VideoLayout.enableDeviceAvailabilityIcons(
                 APP.conference.getMyUserId(), false);
-            VideoLayout.setLocalVideoVisible(false);
+
+            // when we have both iAmRecorder and iAmSipGateway we don't
+            // want local video
```

Comment:
The comment to me says when iAmRecorder and iAmSipGateway are true then disable localVideo. However, the if check are doing if iAmRecorder is true and iAmSipGateway is false then disable local video.

Reply:
Ups, sorry. Does it make more sense now 'in case of iAmSipGateway keep local video visible'. Thanks for reviewing it.

Reply:
Makes sense to me now. Thanks.